package.json

```diff
@@ -6,7 +6,8 @@
   "license": "MIT",
   "scripts": {
     "clean": "rimraf dist",
-    "compile": "npm run clean && bunchee",
+    "build": "npm run clean && bunchee",
+    "compile": "npm run build",
```

Comment:
`npm run build` is way more common than `npm run compile` nowadays. Add `npm run build` and `npm run compile` is now an alias.